docs/dev/RELEASE.md

```diff
@@ -5,14 +5,15 @@
 1. Ensure that you are in a repository that:
    * is clean and up-to-date with no uncommitted changes,
    * was cloned with SSH so that you can push to it, and
-   * has the `master` branch checked out.
+   * has the intended branch checked out (usually `master`,
+     but may be another mainline branch such as `main-3.x`).
 
 1. Select a version number `$VER` (e.g., "3.0.0" or "3.0.0-alpha"). 
    The `major`.`minor`.`patch` numbers may already have been
    incremented since the last release, so they do not necessarily need
    to be updated. However, you may want to increment them further
    depending the types of changes that are in the release.
-1. Run `Scripts/prepare_release.py $VER prepare` from the root of the
+1. Run `Scripts/prepare_release.py $VER prepare --source-branch <this branch>` from the root of the
```

Comment:
Should we mention that the argument is optional, and is `master` by default?

Reply:
Dammit `argparse` why didn't you do this for me automatically... :)

Reply:
(bump for this suggestion)

Reply:
Ah I fixed it for the `--help` of prepare_release.py itself but not here, fixed now.